src/item.cpp

```diff
@@ -9278,16 +9278,24 @@ void item::set_itype_variant( const std::string &variant )
 std::string item::variant_description() const
 {
     if( !has_itype_variant() ) {
-        return "";
+        return ""; // Shouldn't this throw an error?
     }
-
-    // append the description instead of fully overwriting it
-    if( itype_variant().append ) {
-        return _( string_format( "%s  %s", type->description.translated(),
-                                 itype_variant().alt_description.translated() ) );
```

Comment:
@BrettDong Was there any point to this return value being exposed to translation? The text changes thing you pointed out doesn't say anything unexpected is removed now so I'm guessing they weren't getting exposed anyway? Or do the C++ translation changes show up separately?

Reply:
I think this is a code mistake. No point in calling _() on the return value from string format.